util/regexp-assemble/data/include/protocols.data

```diff
@@ -0,0 +1,98 @@
+##! This is a data file used to generate a regular expression for a CRS rule.
+##! The generation of the regular expression happens with the help of
+##! util/regexp-assemble/regexp-assemble.py.
+##! The ID of the rule in question is part of the file name of this data file.
+##! Read more about the format of this data file and the use of the assembly
+##! script in util/regexp-assemble/README.md.
+##!
+##! Lines starting with `##!` are comments and will be skipped,
+##! empty lines will be ignored completely.
+##! In addition, the quote character `'` at the beginning of a line will
+##! cause the line to be interpreted as literal by the cmdline preprocessor only.
+##!
+##! Five special comments are at your disposal to influence the assembled expression:
+##! - `##!+`: the flag comment
+##! - `##!^`: the prefix comment
+##! - `##!$`: the suffix comment
+##! - `##!>`: the preprocessor comment
+##! - `##!<`: the block preprocessor end comment
+##!
+##! Currently supported preprocessors:
+##! - cmdline [windows | unix] (file scope)
+##! Please refer to util/regexp-assemble/README.md for a full explanation.
```

Comment:
This is the new header that is currently waiting to be merged:
```
##! This is a data file used to generate a regular expression for a CRS rule.
##! The generation of the regular expression happens with the help of
##! util/regexp-assemble/regexp-assemble.py.
##! The ID of the rule in question is part of the file name of this data file.
##! Read more about the format of this data file and the use of the assembly
##! script in util/regexp-assemble/README.md.
##!
##! Lines starting with `##!` are comments and will be skipped,
##! empty lines will be ignored completely.
##! In addition, the quote character `'` at the beginning of a line will
##! cause the line to be interpreted literally by the cmdline preprocessor only.
##!
##! Seven special comments are at your disposal to influence the assembly of the
##! expression:
##! - `##!+`: the flag comment
##! - `##!^`: the prefix comment
##! - `##!$`: the suffix comment
##! - `##!>`: the preprocessor comment
##! - `##!<`: the preprocessor end comment
##! - `##!=>`: the concatenation output comment
##! - `##!=<`: the concatenation input comment
##!
##! Currently supported preprocessors:
##! - assemble
##! - cmdline
##! - template
##! - include
##!
##! Please refer to util/regexp-assemble/README.md for a full explanation.
```

Reply:
Oh, ok. Will wait for that one then.

Reply:
Actually, I meant that you could just put this into your file now and then there won't be any merge conflicts when I merge that PR.

Reply:
This will be moved to a unique file, so resolving.